paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -1124,23 +1125,23 @@ class PaymentSheet internal constructor(
                 ) : RowStyle() {
                     constructor(
                         context: Context,
-                        separatorThicknessDp: Int,
-                        startSeparatorInsetDp: Int,
-                        endSeparatorInsetDp: Int,
+                        @DimenRes separatorThicknessRes: Int,
```

Comment:
Can you add to the changelog with a [BREAKING] tag?

Be sure to note this is part of the `ExperimentalEmbeddedPaymentElementApi`.